src/pages/academy/components/search_banner/_search-bar.tsx

```diff
@@ -0,0 +1,419 @@
+                                style={{ color: 'var(--color-blue-3)' }}
+                                onMouseDown={handleSearchClick}
+                            >{`Search for: ${search_query}`}</SearchResultRows>
+
+                            {data_to_render &&
+                                data_to_render.slice(0, 6).map((post, idx: number) => {
+                                    const icon = post.blog_title ? ArticleIcon : VideoIcon
+                                    const icon_alt = post.blog_title ? 'article icon' : 'video icon'
+                                    const handleMouseDown = (e: React.KeyboardEvent) => {
+                                        e.preventDefault()
+                                        navigate(redirect_link_arr[idx])
+                                    }
+
+                                    return (
+                                        <SearchResultRows
+                                            key={post.blog_title || post.video_title}
+                                            jc="flex-start"
+                                            ai="center"
+                                            onMouseDown={handleMouseDown}
+                                            is_active={focus_index === idx}
+                                        >
+                                            {
+                                                <>
+                                                    <IconWrapper src={icon} alt={icon_alt} />
+                                                    <Header
+                                                        as="h3"
+                                                        type="paragraph-1"
+                                                        weight="normal"
+                                                        ml="8px"
+                                                        pt="4px"
+                                                    >
+                                                        {getResultTitles(post)}
+                                                    </Header>
+                                                </>
+                                            }
+                                        </SearchResultRows>
+                                    )
+                                })}
+                        </SearchSuggestionWrapper>
+                    </DesktopWrapper>
+                )}
+            </FormWrapper>
+            <MobileWrapper>
+                <SearchSuggestionWrapper max_width="100%" fd="column">
+                    {search_query && (
+                        <SearchResultRows
+                            jc="flex-start"
+                            ai="center"
+                            is_active={focus_index === -1}
+                            style={{ color: 'var(--color-blue-3)' }}
+                        >{`Search for: ${search_query}`}</SearchResultRows>
```

Comment:
don't we need translation for this part?

Reply:
We only support EN in academy.